client/components/flag/index.jsx

```diff
@@ -15,11 +15,16 @@ const Flag = React.createClass( {
 	render() {
 		const props = omit( this.props, [ 'icon', 'type', 'className' ] );
 
+		var noticon;
```

Comment:
Also, see #1603. We'll probably absorb the `Flag` component into `Notice compact` which already support gridicons and would fix this.